packages/test/test-end-to-end-tests/package.json

```diff
@@ -25,7 +25,7 @@
     "lint:fix": "npm run eslint:fix",
     "start:tinylicious:test": "tinylicious > tinylicious.log 2>&1",
     "test": "npm run test:realsvc",
-    "test:realsvc": "npm run test:realsvc:local && npm run test:realsvc:tinylicious",
+    "test:realsvc": "concurrently npm:test:realsvc:local npm:test:realsvc:tinylicious",
```

Comment:
i don't trust the && here. i think we could be passing when we shouldn't, so trying concurrently while we wait for @curtisman fix. however, that may block on integrate if it turns out always running t9s fails